src/gui/tray/UserModel.cpp

```diff
@@ -881,10 +891,14 @@ QVariant UserModel::data(const QModelIndex &index, int role) const
         return _users[index.row()]->name();
     } else if (role == ServerRole) {
         return _users[index.row()]->server();
+    } else if (role == ServerUserStatusRole) {
```

Comment:
@camilasan Shouldn't we name this `ServerUserStatusRole` and corresponding string `serverUserStatus` to something that also contains a verb `has` as right now it looks like some sort of status from the QML's perspective, while, in fact, it is just a bool within the business logic `serverHasUserStatus()`?

Reply:
I can do that.